db/seeds.rb

```diff
@@ -1,7 +1,8 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rails db:seed command (or created alongside the database with db:setup).
-#
-# Examples:
-#
-#   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
-#   Character.create(name: 'Luke', movie: movies.first)
+Sponsor.delete_all
+
+for i in (1..6) do
+Sponsor.create ({
+  company_name: Faker::Company.name,
+  logo: "http://lorempixel.com/100/50/"
+  })
+end
```

Comment:
I personally don't like this way of creating records and rather add them in an array passed into the `create` method. But fine for now if you like this 😄